### ISSUE TITLE
project.pbxproj diff between 0.63.3 and 0.63.4

### ISSUE THREAD

**Issue by SaeedZhiany** [react-native-community/upgrade-helper#255]
# Question

I have always a problem with this file when I want to upgrade my project. is it necessary to apply the diff shown for the file between  0.63.3 and 0.63.4?

it seems just the hash numbers has been changed (due to random hash generation when installing pods). is there any other changes that I couldn't catch in the file?

Thanks

**Comment by SaeedZhiany** [comment]
Is there any better way to apply the changes for this file specifically? its diff always makes me confused when I want to upgrade my projects.

**Comment by SrBrahma** [comment]
Did you find out how to upgrade it properly?

**Comment by edreyyo** [comment]
What's the recommended approach here, @pvinis? `upgrade-helper` is super useful but the diffs between `*.pbxproj` files are often hard to follow and confusing. Thanks

**Comment by pvinis** [comment]
@edreyyo sorry for the LATE response, but heres a little part of the video https://youtu.be/fmh_ZGHh_eg?t=1368.

In summary, that file usually does not need any changes. if the diff involves anything about pods, you can skip the upgrade helper and just commit the changes that have you have in your repo after `pod install`. If there are other files involved, it will be changes in xcode that you made, or a new file added in xcode, in which case usually we have notes about how to add/remove that file from within xcode.

Please ping me if you have any other issues.
